frontend/src/components/details/TableComponent.jsx

```diff
@@ -16,7 +16,7 @@ class TableComponent extends Component {
     render() {
         const idKey = this.props.tableKey;
         return (
-            <Container fluid={true}>
+            <Container fluid={true} className='TableCorner'>
```

Comment:
This name is not matching with the css name. I will fix that shortly.

Reply:
Thanks @Waldleufer 😄 I probably missed that one somehow

Reply:
done. But it does not affect the table. the corners still are sharp ._.